styles.css

```diff
@@ -1,12 +1,15 @@
 * {
```

Comment:
Kindly follow the design guideline on Figma and fix the font size and font-family issue. To use a font family, first you need to import or download to your repository. You can use the import method on this [link](https://fonts.google.com/specimen/Inter#styles) for Inter font for this section. 
<img width="454" alt="Screenshot 2023-04-04 at 9 23 21 PM" src="https://user-images.githubusercontent.com/25887253/229885573-cdc65499-3d59-49f6-ac26-df7b13225f1a.png">
<img width="600" alt="Screenshot 2023-04-04 at 9 24 31 PM" src="https://user-images.githubusercontent.com/25887253/229885593-0467fc84-1d82-4cb6-9164-eeeded576e04.png">

Reply:
I did import the suggested font but it looks different from whats in the template
![Kazam_screenshot_00083](https://user-images.githubusercontent.com/86020057/229893174-5838ae42-d60b-4835-8dcc-817ea0ec2144.png)

Reply:
Hello @GeekyHacks. You are importing 'Lato' font. Check the GIF below.
![kkk](https://user-images.githubusercontent.com/25887253/229898480-823dd056-3315-46ef-b120-5c5e2779609b.gif)